internal/doctomarkdown/doctomarkdown.go

```diff
@@ -0,0 +1,296 @@
+// Package doctomarkdown converts data in the Sourcegraph LSIF documentation extension format into
+// Markdown format.
+package doctomarkdown
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/sourcegraph/sourcegraph/enterprise/lib/codeintel/lsif/protocol"
+	"github.com/sourcegraph/sourcegraph/enterprise/lib/codeintel/lsif/protocol/reader"
+)
+
+// Convert converts the LSIF data in r to Markdown by scanning for data in the Sourcegraph LSIF
+// documentation extension format.
+func Convert(ctx context.Context, r io.Reader, matchingTags []protocol.DocumentationTag) (string, error) {
```

Comment:
It looks like this entire package is used only for testing. Do you plan to make this available more generally? If not, I'd probably make this an internal test file in the indexer package. If you do make it more available generally (sharing code with the internal conversion process), then that code should be public and in sg/sg/lib.

Reply:
just for testing, will move.